app.js

```diff
@@ -210,6 +210,9 @@ app.get('/api/v1/geo', cors(), resources.v1.geo.get)
 
 app.get('/services/geoip', resources.services.geoip.get)
 
+app.options('/services/cloudinary', cors())
+app.get('/services/cloudinary', cors(), resources.services.cloudinary.get)
```

Comment:
Let's rename the path to something like `/services/images` so that it's not as closely coupled with the third party service we use.

Reply:
Should I also rename the file handling the GET request to images? so `resources.services.images.get`?

Reply:
That seems to be more consistent with existing naming, so I'd say yes.

Reply:
Yup, just checking. Renamed in latest commit!